environments/staging/terraform.yml

```diff
@@ -211,10 +211,10 @@ servers:
     group: "couchdb2"
     os: jammy
 
-  - server_name: "couch26-staging"
+  - server_name: "couch27-staging"
     server_instance_type: "t3a.large"
     network_tier: "db-private"
-    az: "a"
```

Comment:
Just curious what this setting does.

Reply:
This sets the availability zone for the machine https://aws.amazon.com/about-aws/global-infrastructure/regions_az/